x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/privileges/get_missing_privileges.ts

```diff
@@ -38,13 +38,14 @@ export const registerSiemRuleMigrationsGetMissingPrivilegesRoute = (
             const core = await context.core;
             const securitySolution = await context.securitySolution;
             const siemClient = securitySolution?.getAppClient();
+            const spaceId = securitySolution?.getSpaceId();
             const esClient = core.elasticsearch.client.asCurrentUser;
 
             if (!siemClient) {
               return response.notFound();
             }
 
-            const lookupsIndexPattern = `${LOOKUPS_INDEX_PREFIX}*`;
+            const lookupsIndexPattern = `${LOOKUPS_INDEX_PREFIX}${spaceId}*`;
```

Comment:
also open to whatever index pattern you want, seems like the only convention is to pick something ha, alerts uses alerts-{spaceName}, but plenty of other things use underscores as well

Reply:
Yes better separate with underscores `${LOOKUPS_INDEX_PREFIX}_${spaceId}*`. The reason we don't use hyphen is the ESQL LOOKUP syntax, it throws an error when using it, we need to use underscores:
```| LOOKUP JOIN 'index_name' ON 'field_to_match'```